multisig-wallet/project/multisig-contract/src/interface.sw

```diff
@@ -62,25 +80,31 @@ abi MultiSignatureWallet {
     ///
     /// # Reverts
     ///
-    /// - When the constructor has not been called to initialize the contract.
-    /// - When the amount of the asset being sent is greater than the balance in the contract.
-    /// - When the public key cannot be recovered from a signature.
-    /// - When the recovered addresses are not in ascending order (0x1 < 0x2 < 0x3...).
-    /// - When the total approval count is less than the required threshold for execution.
+    /// * When the constructor has not been called to initialize the contract.
+    /// * When the amount of the asset being sent is greater than the balance in the contract.
+    /// * When the public key cannot be recovered from a signature.
+    /// * When the recovered addresses are not in ascending order (0x1 < 0x2 < 0x3...).
+    /// * When the total approval count is less than the required threshold for execution.
     #[storage(read, write)]
     fn transfer(asset_id: ContractId, data: b256, signatures: Vec<SignatureInfo>, to: Identity, value: u64);
+}
 
-    /// Returns the current nonce in the contract
-    #[storage(read)]
-    fn nonce() -> u64;
-
+abi Info {
     /// Returns the contract's balance of the specified asset.
     ///
     /// # Arguments
     ///
     /// * `asset_id` - The contract ID of the asset to check that balance of.
     fn balance(asset_id: ContractId) -> u64;
```

Comment:
Yeah, I did have that initially but then I changed it back to alphabetical rather than annotation and alphabetical. The point is mainly around a clear and consistent structure that makes sense rather than a rule so if you'd like then in another PR I can change it to be along the lines of the guide.

Reply:
It's only a nit, I personally prefer ordering by annotation and then alphabetically, but I think things like this should be author's choice